cpp/serve/threaded_engine.cc

```diff
@@ -173,9 +174,11 @@ class ThreadedEngineImpl : public ThreadedEngine {
           }
         } else if (kind == InstructionKind::kDebugCallFuncOnAllAllWorker) {
           CHECK(background_engine_ != nullptr) << "Background engine is not loaded.";
-          background_engine_->DebugCallFuncOnAllAllWorker(Downcast<String>(arg));
+          Array<ObjectRef> packed_args = Downcast<Array<ObjectRef>>(arg);
+          background_engine_->DebugCallFuncOnAllAllWorker(Downcast<String>(packed_args[0]),
```

Comment:
here the packed_args[1] might be None. I'm not sure whether this will cause an error here.